clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -1048,6 +1048,19 @@ def err_ptrauth_extra_discriminator_invalid : Error<
   "invalid extra discriminator flag '%0'; '__ptrauth' requires a value between "
   "'0' and '%1'">;
 
+// __ptrauth qualifier options string
+def note_ptrauth_evaluating_options
+    : Note<"options parameter evaluated to '%0'">;
+def err_ptrauth_invalid_option : Error<"'__ptrauth' options parameter %0">;
```

Comment:
Look for uses of `enum_select` in that file and do something similar here - lets avoid partial diagnostic strings in cpp files